bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
I used -n here so that $config_cmd could be a function defined in bridge-stp.conf instead of an executable file on the filesystem.  I'm OK with changing this, but be aware that changing it to -x would no longer allow a function to be used for $config_cmd.